app/helpers/quadicon_helper.rb

```diff
@@ -151,6 +151,10 @@ def render_quadicon(item, options = {})
     end
   end
 
+  def img_for_physical_vendor(item)
```

Comment:
Could this be moved to a decorator? ([here's an example](https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/decorators/host_decorator.rb)) We're trying to clean up this code and that would help :)